### PR TITLE
feat(row): add support for space-evenly

### PR DESCRIPTION
**Pull request by gp5251**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
add support for space-evenly

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  add support for space-evenly |
| 🇨🇳 Chinese |  支持space-evenly  |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/grid/demo/flex.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ title:
 
 布局基础。
 
-子元素根据不同的值 `start`,`center`,`end`,`space-between`,`space-around`，分别定义其在父节点里面的排版方式。
+子元素根据不同的值 `start`,`center`,`end`,`space-between`,`space-around`,`space-evenly`，分别定义其在父节点里面的排版方式。
 
 ## en-US
 
-Child elements depending on the value of the `start`,`center`, `end`,`space-between`, `space-around`, which are defined in its parent node typesetting mode.
+Child elements depending on the value of the `start`,`center`, `end`,`space-between`, `space-around`, `space-evenly`, which are defined in its parent node typesetting mode.
```
Comment (afc163) [inline components/grid/demo/flex.md:16]:
```suggestion
Child elements depending on the value of the `start`, `center`, `end`, `space-between`, `space-around` and `space-evenly`, which are defined in its parent node typesetting mode.
```

File: components/grid/demo/flex.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ title:
 
 布局基础。
 
-子元素根据不同的值 `start`,`center`,`end`,`space-between`,`space-around`，分别定义其在父节点里面的排版方式。
+子元素根据不同的值 `start`,`center`,`end`,`space-between`,`space-around`,`space-evenly`，分别定义其在父节点里面的排版方式。
```
Comment (afc163) [inline components/grid/demo/flex.md:12]:
```suggestion
子元素根据不同的值 `start`、`center`、`end`、`space-between`、`space-around` 和 `space-evenly`，分别定义其在父节点里面的排版方式。
```

**Comment by afc163** [comment]
Run `npm test -- components/grid -u` and commit it to update demo snapshot.

**Comment by afc163** [comment]
feature branch please.

**Comment by gp5251** [comment]
do you mean switch to feature branch and edit again? or just merge to feature from master?

**Comment by afc163** [comment]
I mean switch to feature, new feature should be sent to feature branch.

**Comment by gp5251** [comment]
ok. i'll pr again. this is closed.

**Pull request closed without merging**